semgrep/tests/e2e/snapshots/test_autofix/test_regex_autofix/rulesautofixflask-use-jsonify.yaml-autofixflask-use-jsonify.py/results.json

```diff
@@ -17,7 +17,25 @@
         "lines": "    return json.dumps(user_dict)",
         "message": "flask.jsonify() is a Flask helper method which handles the correct settings for returning JSON from Flask routes",
         "metadata": {},
-        "metavars": {},
+        "metavars": {
```

Comment:
just curious, why semgrep-old was not putting the metavars here?

Reply:
I'm not actually sure, @brendongo do you know?